application/src/main/resources/locale/blueprint/horizons/category.csv

```diff
@@ -1,9 +1,9 @@
 messageKey,,en,de,es,fr,pt,ru
-blueprint.category.name.core_internal,Core Internals,Core Internals,Intern (Basis),Internales núcleos,Noyau internes,Principais internos,Основное оборудование
+blueprint.category.name.core_internal,Core Internals,Core Internals,Intern (Basis),Internales núcleos,Interne primordiaux,Principais internos,Основное оборудование
 blueprint.category.name.engineer_unlocks,Engineer Unlocks,Engineer Unlocks,Ingenieur-Freischaltungen,Desbloqueo del ingeniero,"Ingénieurs, Déverrouillage",Desbloqueios do engenheiro,Разблокировка инженеров
 blueprint.category.name.experimental_effects,Experimental Effects,Experimental Effects,Experimentelle Effekte,Efectos experimentales,Effets expérimentaux,Efeitos experimentais,Экспериментальные эффекты
-blueprint.category.name.hardpoint,Hardpoints,Hardpoints,Waffenaufhängungen,Puntos duros,Points durs,Hardpoints,Гнёзда
-blueprint.category.name.optional_internal,Optional Internals,Optional Internals,Intern (Optional),Internales opcionales,Internes facultatifs,Internos opcionais,Доп. Оборудование
+blueprint.category.name.hardpoint,Hardpoints,Hardpoints,Waffenaufhängungen,Puntos duros,Points d’emport,Hardpoints,Гнёзда
```

Comment:
use 2 single quotes `d''emport`, otherwise it doesn't show up correctly inside the app

Reply:
this also applies to all other files, but I won't repeat it everywhere